README.txt
Tesseract Data Consumer based on HTML templates. Refer to the manual for more details.

ChangeLog
2013-06-21 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.6.0

2013-06-19 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added FILES object for FAL support in output, resolves #49262
	* Expanded soft reference parser to read ###RECORD### markers, resolves #49180

2013-06-12 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added support for FAL in templates, including soft reference parser, resolves #47150

2013-05-22 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Verified compatibility with TYPO3 CMS 6.1, resolves #47516

2013-04-12 Roberto Presedo (Cobweb)  <typo3@cobweb.ch>

	* Expanded PAGE_STATUS feature to use page id's, resolves #43128

2013-04-12 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Made full usage of centralized debug and messaging API, resolves #38397
	* Verified compatibility with TYPO3 CMS 6.0, resolves #47149
	* Allowed ### to be displayed in content using escaping, resolves #41993

2013-04-11 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Used new API for accessing controller properties, resolves #47130

2013-01-07 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Corrected error in fetching related controller in BE mapping interface, resolves #44364
	* Released to TER as version 1.5.2

2012-12-20 Roberto Presedo (Cobweb)  <typo3@cobweb.ch>

	* Corrected code to stop execution after redirection, resolves #44158

2012-09-18 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Corrected forcing useCacheHash configuration, resolves #41039
	* Released to TER as version 1.5.1

2012-07-10 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.5.0

2012-06-29 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added definition of search fields, resolves #38487

2012-06-26 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Cleaned up some code, verified compatibility with TYPO3 4.7, resolves #38396

2012-06-07 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.4.0

2012-05-16 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added a MEDIA element type, resolves #37211

2012-04-19 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.3.0

2012-04-03 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Dropped usage of tx_tesseract_utilities::wrapMessage(), resolves #35602
	* Added CSH for tx_templatedisplay_displays table, resolves #9250

2012-03-21 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Fixed bug when resetting the component (reinitialized object list), resolves #35098
	* Released to TER as version 1.2.1

2012-02-22 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Changed static TS to use reference to lib.parseFunc_RTE, resolves #34184

2012-02-06 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.2.0

2012-01-08 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Replaced reference to parent object with new, general member variable, resolves #33051

2011-12-09 Francois Suter (Cobweb)  <typo3@cobweb.ch>

    * Added a RECORDS element type, resolves #32430
    * Updated the manual, added an "Installation" chapter, resolves #13483

2011-12-06 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Improved the PAGE_STATUS function, resolves #32327
	* Corrected spelling mistake in file of class tx_templatedisplay_ajax, resolves #32335
	* Added missing documentation about EMPTY structure, resolves #32336

2011-08-27 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Image resource type should not override configured "file" property, resolves #29302

2011-08-16 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.1.0

2011-07-27 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Image type should not override configured "file" property, resolves #28520

2010-09-27 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* First public release

ext_emconf.php
<?php

/*********************************************************************
* Extension configuration file for ext "templatedisplay".
*
* Generated by ext 21-06-2013 12:56 UTC
*
* https://github.com/t3elmar/Ext
*********************************************************************/

$EM_CONF[$_EXTKEY] = array (
  'title' => 'HTML-based Data Consumer - Tesseract project',
  'description' => 'Use HTML-based templates to display any kind of data returned by a Data Provider, thanks a to user-friendly mapping interface. More info on http://www.typo3-tesseract.com',
  'category' => 'fe',
  'author' => 'Francois Suter (Cobweb) / Fabien Udriot',
  'author_email' => 'typo3@cobweb.ch',
  'shy' => '',
  'dependencies' => 'tesseract',
  'conflicts' => '',
  'priority' => '',
  'module' => '',
  'state' => 'stable',
  'internal' => '',
  'uploadfolder' => 0,
  'createDirs' => '',
  'modify_tables' => '',
  'clearCacheOnLoad' => 0,
  'lockType' => '',
  'author_company' => '',
  'version' => '1.6.0',
  'constraints' => 
  array (
    'depends' => 
    array (
      'typo3' => '4.5.0-6.1.99',
      'tesseract' => '1.7.0-0.0.0',
    ),
    'conflicts' => 
    array (
    ),
    'suggests' => 
    array (
    ),
  ),
  '_md5_values_when_last_written' => 'a:59:{s:9:"ChangeLog";s:4:"9bfc";s:28:"class.tx_templatedisplay.php";s:4:"0954";s:33:"class.tx_templatedisplay_ajax.php";s:4:"d201";s:37:"class.tx_templatedisplay_tceforms.php";s:4:"106a";s:16:"ext_autoload.php";s:4:"ff70";s:21:"ext_conf_template.txt";s:4:"f2bd";s:12:"ext_icon.gif";s:4:"e0c8";s:12:"ext_icon.png";s:4:"aaa6";s:17:"ext_localconf.php";s:4:"c9cc";s:14:"ext_tables.php";s:4:"1fa5";s:14:"ext_tables.sql";s:4:"801d";s:36:"icon_tx_templatedisplay_displays.png";s:4:"3dbb";s:10:"README.txt";s:4:"db38";s:7:"tca.php";s:4:"5aee";s:15:"wizard_icon.gif";s:4:"3075";s:70:"Resources/Private/Language/locallang_csh_txtemplatedisplaydisplays.xml";s:4:"48dd";s:43:"Resources/Private/Language/locallang_db.xml";s:4:"5eee";s:39:"Resources/Private/Snippets/snippets.xml";s:4:"b3fe";s:48:"Resources/Private/Templates/templatedisplay.html";s:4:"496c";s:34:"Resources/Public/images/accept.png";s:4:"036a";s:33:"Resources/Public/images/email.png";s:4:"e7f6";s:33:"Resources/Public/images/empty.png";s:4:"81a2";s:33:"Resources/Public/images/error.png";s:4:"c847";s:39:"Resources/Public/images/exclamation.png";s:4:"1ee9";s:33:"Resources/Public/images/image.png";s:4:"c2fe";s:41:"Resources/Public/images/imageResource.png";s:4:"d928";s:32:"Resources/Public/images/link.png";s:4:"49be";s:40:"Resources/Public/images/linkToDetail.png";s:4:"49be";s:38:"Resources/Public/images/linkToFile.png";s:4:"49be";s:38:"Resources/Public/images/linkToPage.png";s:4:"49be";s:35:"Resources/Public/images/loading.gif";s:4:"9c92";s:47:"Resources/Public/images/mappings_screenshot.png";s:4:"b9fb";s:33:"Resources/Public/images/media.png";s:4:"5ad1";s:41:"Resources/Public/images/missing_image.png";s:4:"279d";s:38:"Resources/Public/images/paintbrush.png";s:4:"247e";s:34:"Resources/Public/images/pencil.png";s:4:"a34e";s:31:"Resources/Public/images/raw.png";s:4:"dfcf";s:35:"Resources/Public/images/records.png";s:4:"655c";s:36:"Resources/Public/images/richtext.png";s:4:"97d0";s:36:"Resources/Public/images/tag_blue.png";s:4:"0824";s:37:"Resources/Public/images/tag_green.png";s:4:"8205";s:38:"Resources/Public/images/tag_orange.png";s:4:"97ce";s:36:"Resources/Public/images/tag_pink.png";s:4:"30f8";s:38:"Resources/Public/images/tag_purple.png";s:4:"af7c";s:35:"Resources/Public/images/tag_red.png";s:4:"5d63";s:38:"Resources/Public/images/tag_yellow.png";s:4:"c8a1";s:32:"Resources/Public/images/text.png";s:4:"2039";s:35:"Resources/Public/images/unknown.png";s:4:"b6ee";s:32:"Resources/Public/images/user.png";s:4:"3d3c";s:42:"Resources/Public/javascripts/formatJson.js";s:4:"23f4";s:47:"Resources/Public/javascripts/templatedisplay.js";s:4:"224b";s:48:"Resources/Public/stylesheets/templatedisplay.css";s:4:"bff5";s:14:"doc/manual.pdf";s:4:"03e9";s:14:"doc/manual.sxw";s:4:"0ddc";s:14:"doc/manual.txt";s:4:"eb14";s:54:"interfaces/interface.tx_templatedisplay_customtype.php";s:4:"54b4";s:41:"samples/class.tx_templatedisplay_hook.php";s:4:"24a5";s:46:"samples/class.tx_templatedisplay_phonetype.php";s:4:"b146";s:16:"static/setup.txt";s:4:"d038";}',
  'suggests' => 
  array (
  ),
  'user' => 'francois',
  'comment' => 'Compatibility with TYPO3 6.0 and 6.1; FAL support; some small new features.',
);

?>